test/common/lib/controller/encrypt.controller-test.js

```diff
@@ -0,0 +1,209 @@
+'use strict';
+
+define(function(require) {
+
+  var sub = require('common/lib/controller/sub.controller');
+  var EncryptController = require('common/lib/controller/encrypt.controller').EncryptController;
```

Comment:
I gotten the unit tests to work with the current AMD dependency tree. Not changes to the src were necessary ...